bundles/org.openhab.binding.loxone/src/main/java/org/openhab/binding/loxone/internal/types/LxIRCTemperature.java

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.loxone.internal.types;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.smarthome.core.library.types.DecimalType;
+import org.eclipse.smarthome.core.types.State;
+import org.eclipse.smarthome.core.types.UnDefType;
+
+/**
+ * Intelligent Room Controller Temperature.
+ *
+ * @author Pawel Pieczul - initial contribution
+ *
+ */
+@NonNullByDefault
+public enum LxIRCTemperature {
+    ECONOMY(0),
+    COMFORT_HEATING(1),
+    COMFORT_COOLING(2),
+    EMPTY_HOUSE(3),
+    HEAT_PROTECTION(4),
+    INCREASED_HEAT(5),
+    PARTY(6),
+    MANUAL(7),
+    UNKNOWN(8);
+
+    private Integer index;
+
+    private LxIRCTemperature(Integer index) {
+        this.index = index;
+    }
+
+    public static LxIRCTemperature fromIndex(Integer index) {
+        for (LxIRCTemperature t : LxIRCTemperature.values()) {
+            if (t.index.equals(index)) {
+                return t;
+            }
+        }
+        return UNKNOWN;
+    }
+
+    public String getLabel() {
+        switch (index) {
+            case 0:
+                return "Economy";
```

Comment:
This is all static so it should be a parameter of the enum: `ECONOMY(0, "Economy")`